### ISSUE TITLE
Server getting random crashes since 7/6/2018 CSGO Update

### ISSUE THREAD

**Issue by vtrbh** [splewis/csgo-pug-setup#222]
If reporting a bug, fill in the following sections:

### Expected behavior
Run matches without server crashing.

### Actual behavior
Server getting random crashes since **[7/6/2018 CSGO Update](http://blog.counter-strike.net/index.php/2018/07/20556/)**
(Never had those crashes before updating the server to current CSGO version)
I run this in a LAN that runs lots of matches a day, and no one uses commands mid-game, just normally plays the pug. Can't seem to find a reason for them. Is there any way to get accurate logs for it to help you fixing it?

### Steps to reproduce
- Plugin version: **2.0.3**
- Sourcemod version: **stable 1.9 - build 6244**
- Steps to reproduce (please be specific): since they occurred at **random** times without any command being used at the moment of it, i couldn't force reproduce it

**Comment by Technoblazed** [comment]
Install https://forums.alliedmods.net/showthread.php?t=277703& and get a crash log the next time it crashes, this probably isn't anything to do with pugsetup.

**Comment by vtrbh** [comment]
I don't use any other plugin neither changed any of the server settings or pug-setup settings, but i will install it and take a look at the results. Thanks @Technoblazed

**Comment by CJNiiva** [comment]
I got the same problem, it seems to happen everytime I do !setup and try to start it

**Comment by Technoblazed** [comment]
Do the same steps I posted earlier please.

**Comment by CJNiiva** [comment]
fun fact, it doesnt make the server crash, it makes the game crash when panorama is enabled

**Comment by CJNiiva** [comment]
dammit, thought it was a problem with the server

**Comment by Technoblazed** [comment]
TLDR; not a pug-setup issue then, it's a community server issue

**Comment by CJNiiva** [comment]
no, it happens just when I press finish setup

**Comment by Technoblazed** [comment]
What's your sourcemod version? That's already been patched

**Comment by CJNiiva** [comment]
1.8.0.6048

**Comment by CJNiiva** [comment]
I'm so sorry for bothering you, I can see that they have released 1.9:/

**Comment by Technoblazed** [comment]
You need to use 1.9.0.6241 and later for CS:GO, any older versions are no longer supported.

**Comment by Technoblazed** [comment]
Install accelerator (linked above) and get a crash log

**Comment by splewis** [comment]
I'm going to close this for now since I've seen no evidence (or a crash dump) that leads me to believe this is related to pugsetup.
